Nasal/mooring.nas

```diff
@@ -8,10 +8,9 @@
 #   COPYRIGHT
 ############################
 #init Moorage.
-#Replace  the generic Seaboat fightgear system which is wrong, and not completed with the list of harbours.
-#This is a customized developpement done from an original idea =>  the Boeing 314A Project ( the nice and accurate FG Clipper) thanks to his unknown Author.
-
-#We are using the original harbour list which was used by the boeing314 model and using too a specific customized addon list which could be improved .
+#Replace the generic Seaboat fightgear system which is wrong, and not completed with the list of harbours.
+#This is a customized developpement done from an original idea => the Boeing 314A Project (the nice and accurate FG Clipper) thanks to his unknown Author.
```

Comment:
development

Reply:
Yeah, I saw that. I was reluctant to change the "original" copy as I copied it over from the aircraft the code originated from.

Reply:
ok... my OCD would probably fix it and send them a patch... all without my knowledge, of course :laughing: